testnet/shared-config-test.json

```diff
@@ -117,6 +117,22 @@
           "address": "0xAE3283fb214cDb14884039Df09B7895773e68eFA",
           "symbol": "PHA",
           "decimals": 18
+        },
+        {
+          "resourceId": "0x0000000000000000000000000000000000000000000000000000000000001200",
+          "caip19": "eip155:11155111/erc20:0x1c7D4B196Cb0C7B01d743Fbc6116a902379C7238",
+          "type": "fungible",
+          "address": "0x1c7D4B196Cb0C7B01d743Fbc6116a902379C7238",
+          "symbol": "USDC",
+          "decimals": 6
+        },
+        {
+          "resourceId": "0x0000000000000000000000000000000000000000000000000000000000002000",
```

Comment:
This resourceID already exists (tTNT token)

Reply:
nooooooo :cry: